LoRaEngine/LoraKeysManagerFacade/IoTCentralImp/Definitions/Device.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade.IoTCentralImp.Definitions
+{
+    public class Device
```

Comment:
Seal:

```suggestion
    public sealed class Device
```

Also, can we make this class immutable?

Reply:
Ditto for all other types being introduced in this namespace.